merlin-sdk/src/main/java/gov/nasa/jpl/aerie/merlin/protocol/types/ValueSchema.java

```diff
@@ -195,6 +195,19 @@ public static ValueSchema ofVariant(final List<Variant> variants) {
     };
   }
 
+  /**
+   * Creates a {@link ValueSchema} representing the Unit type.
+   *
+   * @return A new {@link ValueSchema} representing the Unit type.
+   */
+  public static ValueSchema ofUnit() {
+    return new ValueSchema() {
+      public <T> T match(final Visitor<T> visitor) {
+        return visitor.onVariant(List.of(new Variant("Unit", "UNIT")));
+      }
+    };
+  }
+
```

Comment:
My understanding of `Unit` is that it is a placeholder to fill the `Optional` returned from each `asType()` method. `Unit` itself isn't meaningful, so I'm surprised to see this change. I admittedly am not fully confident my understanding is whole and correct, but if it is correct then I'm not sure this change makes sense. Let me know what you're thinking here.